### DISCUSSION TITLE
Changelog/release note doesn't contain supported pulsar version

### DISCUSSION THREAD

**Discussion by fobiagwu** [milvus-io/milvus#36893] [Q&A and General discussion]
Hello team.
I noticed that the release notes doesn't contain the supported pulsar version. If someone wants to find what the pulsar versions is compatible with a release, where is this information found?

**Comment by xiaofan-luan** [comment]
right now for 2.4.x we are using pulsar 2.9.
for 2.5 or any release above we use pulsar 3.0
